quadcore/extractor/article.py

```diff
@@ -0,0 +1,20 @@
+from quadcore.extractor import Extractor
+
+class ArticleExtractor(Extractor):
+    @classmethod
+    def preprocess(cls, obj, options):
+        """
+        Preprocess obj to significant {key:value}
+        Returns list of urls to fetch.
+        """
+        entity_dict = dict()
+        key_num = 1
```

Comment:
why do we need this key and the article num? Is the article num being used for duplicate checking? if that's the case, it's a very brittle way of maintaining: if we have a bug that incorrectly assigns the article num, it'll start deduping articles (or incorrectly not deduping them). Typically maintaining these unique id number manually is a headache..

Couldn't we always just get the list of articles in a list (so this method would be returning a list of articles)? Deduping should be done by the URL level.

Reply:
@achaccha Check this comment :)

Reply:
I thought I should do it in a dictionary format only...
I'll change it to list format and match it to url.